sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/ToString.java

```diff
@@ -38,74 +37,67 @@ private ToString() {
   }
 
   /**
-   * Returns a {@code PTransform<PCollection, PCollection<String>>} which transforms each
-   * element of the input {@link PCollection} to a {@link String} using the
-   * {@link Object#toString} method.
+   * Transforms each element of the input {@link PCollection} to a {@link String} using the {@link
+   * Object#toString} method.
    */
-  public static PTransform<PCollection<?>, PCollection<String>> elements() {
-    return new SimpleToString();
+  public static Elements elements() {
```

Comment:
I'm clearly dumb, because I've asked this question in different flavors a few times, but once more for good measure. Sorry!

The last few commits in this PR left inner transforms generic; why is Elements public? Is it for further customization, or another reason for a difference in visibility?